index.html

```diff
@@ -5,11 +5,62 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=assets, initial-scale=1.0">
-    <title>Unspalsh</title>
+    <title>Unsplash</title>
     <link rel="stylesheet" href="./style.css">
 </head>
 
 <body>
+    <nav>
+        <div class="flex justify-between align-center mb-6 mx-4 mt-4 flex-wrap gap-x-1 top-navbar">
+            <div class="flex align center justify-between gap-x-4">
+                <a href="https://unsplash.com/"> <img src="./assets/unsplash (2).webp" alt="logo" width="55"> </a>
+                <form class="relative text-input-with-icon">
+                    <input class="text-input text-input-with-icon search-input" placeholder=" Search" type="text"
+                        name="search">
+                    <svg class="text-input-with-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+                        <path
+                            d="M416 208c0 45.9-14.9 88.3-40 122.7L502.6 457.4c12.5 12.5 12.5 32.8 0 45.3s-32.8 12.5-45.3 0L330.7 376c-34.4 25.2-76.8 40-122.7 40C93.1 416 0 322.9 0 208S93.1 0 208 0S416 93.1 416 208zM208 352c79.5 0 144-64.5 144-144s-64.5-144-144-144S64 128.5 64 208s64.5 144 144 144z" />
+                    </svg>
+                    <input hidden type="submit" value="Search" />
+                </form>
+            </div>
+            <div class="flex align-center justify-between gap-x-4 top-navbar__item">
+                <a href="https://unsplash.com/explore"> Explore </a>
+                <a href="https://unsplash.com/advertise"> Advertise </a>
+                <a href="https://unsplash.com/plus" class="gradient"> Unsplash + </a>
+            </div>
+            <div class="divider top-navbar__item"> </div>
+            <div class="flex align-center justify-around gap-2 top-navbar__item">
+                <a href="https://unsplash.com/login"> Log in </a>
+                <p> / </p>
+                <a href="https://unsplash.com/join"> Sign up </a>
+            </div>
+            <button type="button" class="button-with-border top-navbar__item"> Submit a photo </button>
+            <button type="button" class="button-without-border"> <svg xmlns="http://www.w3.org/2000/svg"
```

Comment:
Why doesn't this item have the `top-navbar__item` class? I would expect all of them (except dividers) to have it.